frontend_tests/node_tests/pm_list.js

```diff
@@ -45,12 +45,11 @@ global.people.initialize_current_user(me.user_id);
 
 (function test_get_conversation_li() {
     var test_conversation = 'foo@example.com,bar@example.com';
-    var error_msg;
-    global.blueslip.warn = function (error) {
-        error_msg = error;
-    };
+    blueslip.set_test_data('warn', 'Unknown conversation: ' + test_conversation);
+    blueslip.set_test_data('warn', 'Unknown emails: ' + test_conversation); // people.js
```

Comment:
In the current setup, now a test for `pm_list` also needs to watch out for a warning thrown by `people`. I think something doesn't feel right here.

1. It wouldn't be correct to disable the current setup's errors on encountering unexpected warnings as that could let bugs slip by.
2. This module shouln't ideally be running the `people.js` code in unit tests. I know we do not write *completely* unit tests, but should we strive towards removing such dependencies in tests? Or do we let these be?